generate/templates.py

```diff
@@ -19,6 +19,21 @@
 //! THIS FILE HAS BEEN GENERATED -- SAVE ANY MODIFICATIONS BEFORE REPLACING.
 
 use async_google_apis_common::*;
+
```

Comment:
I haven't tried it, but is it possible to move this into the async-google-apis-common crate and import it? It is not a huge difference, but would result in there only being one type definition even if using several generated files.

Reply:
You're right, these types need to be in the crate, not template.

Reply:
That is a very good suggestion, I'll update the PR later today.

Reply:
I moved the types to async_google_apis_common.
In addition I:
- made all the non-set methods take &self instead of &mut self (they don't mutate self and doing so removes any need for a lock on the *Service types)
- made "multi-thread" the default in async_google_apis_common
- updated the example crates to use tokio 1.2 and replaced Rc with Arc in both examples.

Reply:
Thanks for your contributions.